README.md

```diff
@@ -36,22 +36,49 @@ different name, run
 $ make container IMAGE=your-repo/your-name:here
 ```
 
-# Deploying the plugins
+## Deploying the plugins
 
 To deploy your plugin image to an Ark server:
 
 1. Make sure your image is pushed to a registry that is accessible to your cluster's nodes.
 2. Run `ark plugin add <image>`, e.g. `ark plugin add gcr.io/heptio-images/ark-plugin-example`
 
-# Using the plugins
+## Using the plugins
+
+***Note***: As of v0.10.0, the Custom Resource Definitions used to define backup and block storage providers has changed. See [the previous docs][3] for using plugins with versions v0.6-v0.9.x.
 
 When the plugin is deployed, it is only made available to use. To make the plugin effective, you must modify your configuration:
-    
-1. Run `kubectl edit config <config-name> -n <ark-namespace>` e.g. `kubectl edit config default -n heptio-ark`
-2. Change the name of section: `backupStorageProvider` for **Object Store** plugin, `persistentVolumeProvider` for **Block Store** plugin
+
+Backup storage:
+
+1. Run `kubectl edit backupstoragelocation <location-name> -n <ark-namespace>` e.g. `kubectl edit backupstoragelocation default -n heptio-ark`
+2. Change the value of `spec.provider` to enable an **Object Store** plugin
 3. Save and quit, the plugin will be used for the next `backup/restore`
 
-# Creating your own plugin project
+1. Run `kubectl edit volumesnapshotlocation <location-name> -n <ark-namespace>` e.g. `kubectl edit volumesnapshotlocation default -n heptio-ark`
+2. Change the value of `spec.provider` to enable a **Block Store** plugin
+3. Save and quit, the plugin will be used for the next `backup/restore`
+
+## Examples
+
+To run with the example plugins, do the following:
+
+1. Run `ark backup-location create --provider file default`
+2. Run `ark snapshot-location create --provider example-blockstore example-default`
```

Comment:
might be slightly clearer to move `example-default` to right after `create`, before the `--provider` flag?

Reply:
Fixed